site/src/layouts/DetailComponent/DetailComponent.tsx

```diff
@@ -64,19 +79,43 @@ export const DetailComponent: FC<LayoutProps> = ({ children }) => {
 
   const currentTabIndex = currentTab?.id ?? 0;
 
+  const {
+    meta: { title },
+  } = useMeta();
+
   return (
     <AllExamplesViewContext.Provider
       value={{ allExamplesView, setAllExamplesView }}
     >
       <DetailBase
         sidebar={
           <Sidebar sticky>
-            {<SecondarySidebar additionalData={useData} />}
+            {
+              <SecondarySidebar
+                additionalData={useData}
+                tableOfContents={<TableOfContents />}
+              />
+            }
           </Sidebar>
         }
+        pageTitle={
+          isMobileView ? (
```

Comment:
both the drawer and the sidebar are showing in the breakpoint that covers 750 - 1000